qemu/tests/ovs_quit.py

```diff
@@ -24,33 +22,33 @@ def run(test, params, env):
 
     def get_ovs_ports(ovs):
         cmd = "ovs-vsctl list-ports %s" % ovs
-        return set(utils.system_output(cmd).splitlines())
+        return set(process.system_output(cmd).splitlines())
 
     os_dep.command("ovs-vsctl")
     netdst = params.get("netdst")
-    if netdst not in utils.system_output("ovs-vsctl list-br"):
-        raise error.TestError("%s isn't is openvswith bridge" % netdst)
+    if netdst not in process.system_output("ovs-vsctl list-br"):
+        test.error("%s isn't is openvswith bridge" % netdst)
 
     deps_dir = data_dir.get_deps_dir("ovs")
-    nic_script = utils_misc.get_path(deps_dir, params["nic_script"])
-    nic_downscript = utils_misc.get_path(deps_dir, params["nic_downscript"])
-    params["nic_script"] = nic_script
-    params["nic_downscript"] = nic_downscript
 
-    params["qemu_command_prefix"] = "export SHELL=/usr/bin/bash;"
     params["start_vm"] = "yes"
     params["nettype"] = "bridge"
-    params["nic_model"] = "virtio-net-pci"
+    params["nic_model"] = "virtio-pci"
```

Comment:
03:58:28 INFO | Running qemu command (reformatted):
...
-device virtio-pci,mac=9a:6b:6c:6d:6e:6f,id=idPXmVYh,vectors=4,netdev=idFUGUH4,bus=pci.0,addr=0x4  \
...
03:58:28 INFO | [qemu output] qemu-kvm: -device virtio-pci,mac=9a:6b:6c:6d:6e:6f,id=idPXmVYh,vectors=4,netdev=idFUGUH4,bus=pci.0,addr=0x4: Parameter 'driver' expects non-abstract device type
....



According to the log above, invalid driver "virtio-pci" is set correctly.